definitions_and_key_concepts.qmd

```diff
@@ -45,7 +45,7 @@ Artificial Intelligence (AI) attempts to simulate human intelligence using techn
 
 ## Black box models {.unnumbered}
 
-Black box models internal workings are not visible, easily understood, or succinctly explained. These models take input and produce output without providing clarity about the process used to arrive at the output. [Artificial Intelligence](#Artificial Intelligence) models (including [Machine Learning](#Machine Learning)) are the most common type of black box models used today. Other forms of black box models may arise in future.  
+Black box models internal workings are not visible, easily understood, or succinctly explained. These models take input and produce output without providing clarity about the process used to arrive at the output, this includes proprietary models with protected intellectual property. [Artificial Intelligence](#Artificial Intelligence) models (including [Machine Learning](#Machine Learning)) can often be considered a type of black box models. Other forms of this type of black box models may arise in future.  
```

Comment:
Suggest a separate sentence for the proprietary models point (which is well made!).  A suggestion: "Proprietary models with protected intellectual property which means methods and processes are not fully transparent are also black boxes."

Reply:
No other comments - I think this looks good apart from my nit picking points :-)